app/assets/javascripts/pagination.js

```diff
@@ -0,0 +1,6 @@
+$(function() {
+  $("#groceries").on("click", "a", function() {
+    $.get(this.href, null, null, "script");
+    return false;
```

Comment:
instead of returning false, accept the event object in the function and call `e.preventDefault()`. Returning `false` is an anti-pattern